traffic_ops/traffic_ops_golang/servercapability/servercapability.go

```diff
@@ -367,3 +367,157 @@ func (v *TOServerCapability) SelectMaxLastUpdatedQuery(where, orderBy, paginatio
 
 func (v *TOServerCapability) Create() (error, error, int) { return api.GenericCreateNameBasedID(v) }
 func (v *TOServerCapability) Delete() (error, error, int) { return api.GenericDelete(v) }
+
+func GetServerCapabilityV5(w http.ResponseWriter, r *http.Request) {
```

Comment:
Rather than copying all of the content from `GetServerCapability` to `GetServerCapabilityV5`, we should be able to use the same function for either and use a type parameter for which `ServerCapability` type to use.